### ISSUE TITLE
Skettchware bug

### ISSUE THREAD

**Issue by Kayr123** [Sketchware-Pro/Sketchware-Pro#508]
I have sketchware latest v 
Everything is working fine expect fonts its just show me a white screen for 1 sec . I can't add fonts please help

**Comment by JavkhlanK** [comment]
Which Sketchware Pro version did you use? Please tell the exact one, not just "latest one".

Can you e.g. show a screen recording of that happening? Does the same issue happen on a newly created project?

**Comment by Kayr123** [comment]
> 
I can't send pictures but yes i tried the same thing on another phone and it works just fine but mine no

**Comment by JavkhlanK** [comment]
You can embed images as well as videos in comments. Click on the slightly darker area under the text field.

**Comment by Kayr123** [comment]
I can't i swear

**Comment by JavkhlanK** [comment]
Weird. Well, can you answer my other questions then, please?

**Comment by Kayr123** [comment]
I used bulid rc1 
Rc4

**Comment by JavkhlanK** [comment]
Tell the exact version name shown in Program information, please.

**Comment by Kayr123** [comment]
Version 6.4.0-rc01-kt

**Comment by Kayr123** [comment]
I really appreciate your help

**Comment by JavkhlanK** [comment]
Can you try using v6.4.0-rc04? It should've fixed Font issues, though only adding them.

**Comment by Kayr123** [comment]
> Can you try using v6.4.0-rc04? It should've fixed Font issues, though only adding them.

Same problem i tried rc04 
And rc03

**Comment by Kayr123** [comment]
It's ok i could find a project with many fonts and then add them to collection if that's possible

**Comment by JavkhlanK** [comment]
So did you get it to work now?

**Comment by Kayr123** [comment]
> So did you get it to work now?

No 😔ُ but it's okay  I'll complete with no fonts

**Comment by JavkhlanK** [comment]
You could upload screen recordings and screenshots to e.g. Google Drive or MediaFire, btw.

**Comment by Kayr123** [comment]
Ohh ok I'll do some vids and pics i thought of that before

**Comment by Kayr123** [comment]
https://www.file-upload.net/download-14972848/ScreenRecorder_20220723-164019.mp4.html the video

**Comment by JavkhlanK** [comment]
![image](https://user-images.githubusercontent.com/40118124/180612395-c9f456a6-9f7c-47ec-a237-960db3e78b62.png)

Not a great service. Got the file though, checking it now.

**Comment by Kayr123** [comment]
I swear i just upload it i have that app called upload net i just upload them like that and anyone can download it like dropox or mediafire 

The video is 38 sec

**Comment by JavkhlanK** [comment]
I'm not sure what's wrong, it works on my phone and apparently most others' too. I think it's an issue with your phone, sorry. What's it, by the way?

**Comment by JavkhlanK** [comment]
You could back up your project as an SWB, send it to your friend, let him add the fonts to the project for you, and make a backup of the project again and send it to you. That way you could work around that bug.

**Comment by Kayr123** [comment]
I already have another phone but I'm to lazy to send the project add the files then send it back :)

**Comment by denzkie18** [comment]
In my case I used this current version. The only problem I encounter is that I cannot add AdMob. It crush the app. and I can't also improt AdMob and Firebase from the other projects. It completely crush the app. But everything is fine except for this.

**Comment by Kayr123** [comment]
Me too i can't import firebase from other projects it crashes the app

**Comment by JavkhlanK** [comment]
@denzkie18 maybe you wanna check out #490, that's its bugtracker.

Where did you find an option to import Firebase details from another project though? Now you can only import them from google-services.json, as the previous logic was broken.

**Comment by Insight-deviler** [comment]
@JavkhlanK bro still the option for importing firebase details from other project exist in the latest release.
It can be found in
 firebase->firebase manager->after enabling firebase in project by importing from Google json->project setting(which is next to question mark In firebase manager)

**Comment by JavkhlanK** [comment]
> @JavkhlanK bro still the option for importing firebase details from other project exist in the latest release. It can be found in firebase->firebase manager->after enabling firebase in project by importing from Google json->project setting(which is next to question mark In firebase manager)

Ah thanks for telling, I wasn't 100 % sure because the import from google-services.json feature's a contribution not by me.

**Comment by Insight-deviler** [comment]
@JavkhlanK but I still don't know why some people use that while they have something which can reduce all the Hassel of importing firebase details

**Comment by denzkie18** [comment]
yep! The firebase and admob is crashing when importing from other project. Hope this will be fixed at new version coming. Thanks for always updating thsi wonderful and powerful application sir. It help me a lot
